build-tools/automation/yaml-templates/use-dot-net.yaml

```diff
@@ -0,0 +1,33 @@
+# This template enables installing master/nighly builds of .NET 5.
```

Comment:
You might update this to say .NET 6 if you change it again.

Reply:
I updated this to be version agnostic and to help explain why we'd want to use it over `UseDotNet`.